package.json

```diff
@@ -136,6 +136,7 @@
 	},
 	"devDependencies": {
 		"@apollo/react-testing": "^3.1.3",
+		"@babel/compat-data": "^7.9.0",
```

Comment:
this is an attempt to fix this error:

![image](https://user-images.githubusercontent.com/1477453/77662998-39d7d100-6f85-11ea-971a-1df001d03d6c.png)

Reply:
based on this article: https://github.com/angular/angular-cli/issues/17262
but this can be wrong